packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/nodejs/integrationsteps.md

```diff
@@ -8,14 +8,13 @@ For example:
 
 <div class="common-image-format">
 
-![Sign in screenshot](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in.png
- "Sign in screenshot")
+![Display of a screenshot showing a simple sign-in page with fields for username and password; a login button submits the credentials.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in-nodejs.png)
```

Comment:
```suggestion
![Display of a screenshot showing a simple sign-in page with fields for username and password. A login button submits the credentials.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in-nodejs.png)
```

Reply:
I like using semicolons too 😄 . But I think our style convention is to avoid them.

Reply:
Thanks. I was wondering for alt-text if we should keep it to a single sentence, but not a big deal. I'm fine with either, but I like to add meaningful alt-text, which I believe is ready by a screen reader.